app/assets/images/index.ts

```diff
@@ -0,0 +1,7 @@
+import image1 from "./Image.png"
+import image2 from "./Image2.png"
```

Comment:
Please use better namings here. When I read the code I don't understand what image1 and image 2 are.